### ISSUE TITLE
AuthenticateAction call fails

### ISSUE THREAD

**Issue by palp** [happyleavesaoc/python-myusps#7]
The call to https://reg.usps.com/entreg/json/AuthenticateAction fails with:
`
{
  "actionErrors": [
    "Our apologies that you are having issues with your login. We recommend that you restart the current browser and try logging in again........  We do not recognize your username and/or password.  Please try again."
  ],
  "actionMessages": [],
  "fieldErrors": {}
}
`
The password is correct (of course) and I've tried two different accounts. Maybe there's been a breaking change on the USPS site?

**Comment by pjmsullivan** [comment]
@palp - I think @happyleavesaoc is aware already.  I believe you are correct that it is a breaking change to the USPS site.  See here: https://community.home-assistant.io/t/usps-sensor-myusps-uspserror-not-logged-in/16434/39?u=pjmsullivan

**Comment by happyleavesaoc** [comment]
I'm looking into this. Haven't found a solution yet. If anyone else solves it before I do, let me know!

**Comment by happyleavesaoc** [comment]
I spent some time looking at this, but it seems to be working again as-is. Can someone confirm that?

**Comment by colinodell** [comment]
I can confirm this issue exists.

When I try running the example code from the README, I'm getting a `KeyError: 'rs'`:

```
Traceback (most recent call last):
  File "test.py", line 6, in <module>
    session = myusps.get_session("colinodell@gmail.com", "XXXXXXXXXXX")
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 237, in get_session
    _login(session)
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 137, in _login
    if resp.json()['rs'] != 'success':
KeyError: 'rs'
```

When I modify the `_login` method to `print(resp.json())` I get the following output:

```
{'actionErrors': ['Our apologies that you are having issues with your login. We recommend that you restart the current browser and try logging in again........  We do not recognize your username and/or password.  Please try again.'], 'actionMessages': [], 'fieldErrors': {}}
```

I also get the same `KeyError` in my HASS error log:

```
2017-06-07 10:16:57 ERROR (MainThread) [homeassistant.helpers.entity] Update for sensor.usps_mail fails
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 155, in wrapped
    return function(*args)
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 205, in get_mail
    container = _require_elem(response, INFORMED_DELIVERY_TAG, INFORMED_DELIVERY_ATTRS)
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 81, in _require_elem
    raise USPSError('Not logged in')
myusps.USPSError: Not logged in

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/app/homeassistant/helpers/entity.py", line 224, in async_update_ha_state
    yield from self.hass.async_add_job(self.update)
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/app/homeassistant/components/sensor/usps.py", line 124, in update
    self._state = len(myusps.get_mail(self._session))
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 157, in wrapped
    _login(args[0])
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 137, in _login
    if resp.json()['rs'] != 'success':
KeyError: 'rs'
```

```
2017-06-07 10:16:58 ERROR (MainThread) [homeassistant.helpers.entity] Update for sensor.usps_packages fails
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 155, in wrapped
    return function(*args)
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 184, in get_packages
    dashboard = _require_elem(response, DASHBOARD_TAG, DASHBOARD_ATTRS)
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 81, in _require_elem
    raise USPSError('Not logged in')
myusps.USPSError: Not logged in

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/app/homeassistant/helpers/entity.py", line 224, in async_update_ha_state
    yield from self.hass.async_add_job(self.update)
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/app/homeassistant/components/sensor/usps.py", line 78, in update
    for package in myusps.get_packages(self._session):
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 157, in wrapped
    _login(args[0])
  File "/usr/local/lib/python3.6/site-packages/myusps/__init__.py", line 137, in _login
    if resp.json()['rs'] != 'success':
KeyError: 'rs'
```

**Comment by happyleavesaoc** [comment]
Thanks for confirming.

I have found a workaround. I will push an update this evening. But I'm afraid it may be temporary.

**Comment by happyleavesaoc** [comment]
Fixed in `1.1.2`.

**Comment by palp** [comment]
Confirmed working for me, thank you!
